packages/svelte/src/lib/component/UploadButton.svelte

```diff
@@ -0,0 +1,263 @@
+<script lang="ts" context="module">
+  // Workaround for eslint-plugin-svelte issue regarding generics:
+  // https://github.com/sveltejs/svelte-eslint-parser/issues/306
+  import type { FileRouter } from "uploadthing/server";
+
+  type TRouter = FileRouter;
+  type TEndpoint = keyof TRouter;
+  type TSkipPolling = boolean;
+</script>
+
+<script
+  lang="ts"
+  generics="TRouter extends FileRouter , TEndpoint extends keyof TRouter, TSkipPolling extends boolean = false"
+>
+  import { onMount } from "svelte";
+  import { twMerge } from "tailwind-merge";
+
+  import {
+    allowedContentTextLabelGenerator,
+    resolveMaybeUrlArg,
+    styleFieldToClassName,
+  } from "@uploadthing/shared";
+  import type { StyleField } from "@uploadthing/shared";
+  import {
+    generateMimeTypes,
+    generatePermittedFileTypes,
+  } from "uploadthing/client";
+
+  import type { UploadthingComponentProps } from "../types";
+  import { INTERNAL_uploadthingHookGen } from "../useUploadThing";
+  import { getFilesFromClipboardEvent, progressWidths } from "./shared";
+  import Spinner from "./Spinner.svelte";
+
+  type ButtonStyleFieldCallbackArgs = {
+    __runtime: "svelte";
+    ready: boolean;
+    isUploading: boolean;
+    uploadProgress: number;
+    fileTypes: string[];
+  };
+
+  type UploadButtonAppearance = {
+    container?: StyleField<ButtonStyleFieldCallbackArgs>;
+    button?: StyleField<ButtonStyleFieldCallbackArgs>;
+    allowedContent?: StyleField<ButtonStyleFieldCallbackArgs>;
+    clearBtn?: StyleField<ButtonStyleFieldCallbackArgs>;
+  };
+
```

Comment:
the `content` prop seems to be missing?

Reply:
or is that what the `<slot>`'s are for? might need some docs for that if so

Reply:
Yes that's what the `<slot>`'s are for, I thought I added that in the docs 🤔 might need to update them